aspnetcore/migration/1x-to-2x/samples/AspNetCoreDotNetFx2.0App/AspNetCoreDotNetFx2.0App/Program.cs

```diff
@@ -20,6 +21,13 @@ public static void Main(string[] args)
         public static IWebHost BuildWebHost(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
+                .ConfigureAppConfiguration((hostContext, config) =>
+                {
+                    config.AddEnvironmentVariables();
```

Comment:
These aren't good ones to show. We should add things that aren't added by default:

```c#
config.AddJsonFile("myconfig.json");
```

Reply:
@davidfowl done... can you advise on resetting the providers?